rtos/source/TARGET_CORTEX/mbed_boot.c

```diff
@@ -64,6 +64,15 @@
 #include "mbed_error.h"
 #include "mbed_mpu_mgmt.h"
 
+#ifdef MBED_CONF_RTOS_PRESENT
```

Comment:
Don't think this is required? This whole directory is RTOS-only code, right?

Reply:
I have not looked at what rtos-api includes (when used in bare metal mode) nor what is used by the bootloader so I wanted to be on the safe side.
I will remove since you seem to imply this source is exclusively used wit te RTOS.

Reply:
It's determined by the mbed_lib.json files. "rtos-api" is the name given in `/rtos/mbed_lib.json`, so requesting rtos-api gives you `/rtos`.

Requesting "rtos" gives you the `/rtos/source/TARGET_CORTEX` directory - the actual RTX implementation.